Assets/PlayroomKit/PlayroomKit.cs

```diff
@@ -37,7 +37,7 @@ public class InitOptions
         private static Action InsertCoinCallback = null;
 
         [DllImport("__Internal")]
-        private static extern void InsertCoinInternal(Action callback, string options, Action<string> onQuitInternalCallback);
+        private static extern void InsertCoinInternal(Action callback, string options, Action<string> onJoinCallBack ,Action<string> onQuitInternalCallback);
```

Comment:
Small nits:
1. Rename `onJoinCallback` to `onPlayerJoinInternalCallback`
2. Fix space between comma and `onJoinCallback`